src/sections/inventory/ingredient-snackbar.tsx

```diff
@@ -5,6 +5,7 @@ interface ResponseSnackbarProps {
   handleCloseSnackbar: () => void;
   severity: SEVERITY;
   message: string;
+  ariaLabel: string;
```

Comment:
`ariaLabel` is a compulsory props, but it's not being used in `ResponseSnackbar`, causing the entire application to not be able to build